fxied typo (#2325)
arbor-sim/arbor: doc/concepts/morphology.rst

Cable cell morphology
=====================

In simulations of morphologically detailed neurons,
`the cell is modelled <https://en.wikipedia.org/wiki/Cable_theory>`_

> as a set of round electrical cables, which have, among other properties,

Reviewer: ```suggestion
as a set of round electrical cables, which have, among others: temperature,
```

> capacitances and resistances. These can in turn be expressed as a system of

Reviewer: ```suggestion
capacitances, and resistances. These can in turn be expressed as a system of
```

  :width: 600
  :align: center

  A ten-segment cable cell, with soma (pink), axon (grey), and dendrite (light blue).
  Python code to generate this cable cell is in the :class:`segment_tree<arbor.segment_tree>`
  documentation :ref:`here <morph-label-seg-code>`.

> * The tree is composed of 11 segments (1 soma, 2 axons, 8 dendrites).

Reviewer: Not sure the plural is best here. The line as written is trying to say:
We have 11 segments, thereof one labeled as the soma, 2 in the axon, and eight in the dendrite.

Note that ``join_at`` and ``split_at`` are inverse to each other.

> A particular use case for these operations is pruning a specific tag region in the

Reviewer: ```suggestion
A particular use case for these operations is pruning a specific region by its tag in the
```
